### PR TITLE
added doc for isXValue()

### PR DESCRIPTION
**Pull request by strikeraryu**
Signed-off-by: strikeraryu <striker.aryu56@gmail.com>

Documentation added for isXValue() as mentioned in Issue #17644.
Docs locally checked using make docs.

### CODE REVIEW

**Comment by bradcray** [comment]
@strikeraryu : Did you ever ask someone to review this PR?  (We often don't notice open PRs because there are so many of them if someone doesn't request a review on them).  Thanks.

**Comment by strikeraryu** [comment]
@bradcray yeah, I think I forgot to mention someone, I only mentioned that I raised a PR. @e-kayrakli please have a look at the PR related to the issue of documentation isXValue().

File: modules/standard/Types.chpl [hunk 1]
```diff
@@ -209,31 +209,31 @@ proc chpl__maxIntTypeSameSign(type t) type {
...
+/* Returns `true` if the argument is a `complex` value. */
 proc isComplexValue(e)   param  return isComplexType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `string` value. */
 proc isStringValue(e)    param  return isStringType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a `bytes` value. */
 proc isBytesValue(e)     param  return isBytesType(e.type);
-pragma "no doc"
+/* Returns `true` if the argument is a value of one the following types: 'int', 'uint'. */
```
Comment (e-kayrakli) [inline modules/standard/Types.chpl:228]: Can you change your single quotes `'` into backticks `` ` ``, and make these lines fit into 80 characters?

**Review by e-kayrakli — CHANGES_REQUESTED** [review]

File: modules/standard/Types.chpl (hunk 1 above)
Comment (strikeraryu) [reply modules/standard/Types.chpl:228]: Change all single quotes right?

Comment (e-kayrakli) [reply modules/standard/Types.chpl:228]: Yes, please. Types should always be like `` `int` ``, and not  `'int'`

Comment (strikeraryu) [reply modules/standard/Types.chpl]: I have made the changes.

Comment (e-kayrakli) [reply modules/standard/Types.chpl]: Why did we separate them with slashes now? Why not `` `int`, `uint` ... `` as it used to be in your initial PR?

Comment (strikeraryu) [reply modules/standard/Types.chpl]: then it will be more than 80 chars

Comment (e-kayrakli) [reply modules/standard/Types.chpl]: You can add a new line :)

Comment (strikeraryu) [reply modules/standard/Types.chpl]: ohh, mb.😅

Comment (strikeraryu) [reply modules/standard/Types.chpl]: Btw while checking after making docs the line is properly aligned (previous one `int`, `uint` ...). should I still fit in 80 chars or just make it as the initial PR.

Comment (e-kayrakli) [reply modules/standard/Types.chpl]: The rendered documentation doesn't need to fit in any number of columns. It is just the source code.

I am not sure if I answered the question you're asking, though.

**Comment by e-kayrakli** [comment]
@strikeraryu -- I think the last commit that you pushed didn't actually change anything.

**Comment by strikeraryu** [comment]
The last commit was the one with `int/unit` and just for clarification, the 80 chars requirements were for the code, not for the builds docs. So something like this would work.
image

**Comment by e-kayrakli** [comment]
Yes and the code that I am seeing here is different than what you have in that screenshot. Screenshot is what we should have but the current code on this PR still has `` `int/uint....` ``

**Comment by strikeraryu** [comment]
I didn't push that time I was waiting for the confirmation, I have pushed now you can have a look.

**Comment by e-kayrakli** [comment]
I confirmed that `make docs` works nicely and hellos pass.

**Review by e-kayrakli — APPROVED** [review]

**Comment by e-kayrakli** [comment]
Thanks @strikeraryu!

**Comment by strikeraryu** [comment]
Great, welcome. @e-kayrakli

**Pull request merged**